src/mobile/apps/profile/index.coffee

```diff
@@ -49,4 +49,5 @@ app.set 'view engine', 'jade'
 app.get(route, routes.setProfile) for route in vanityRoutes
 app.get '/profile/:id', routes.index
 app.get '/editorial', routes.redirectEditorial
+app.get '/collect', routes.redirectCollect
```

Comment:
I wonder if this is still needed seeing that we are loading `../desktop/apps/collect/server` in the index file now?

Reply:
ah you're totally right, forgot to clean those up!